### ISSUE TITLE
[Feature Request] Which component was modified?

### ISSUE THREAD

**Issue by Aircoookie** [jaames/iro.js#140]
Hi James!

Terrific work on v5! I updated a few days ago and I love the new component system, especially the kelvin slider.

Now to my question: Is there a way to find out in the `input:end` event which component the user interacted with?
This info would be very useful (In my case I would like the application to behave differently to a Kelvin value than to an RGB color).

*Alternatives considered*
I guess I could just use two separate pickers, but that would complicate the application logic compared to using a single picker with multiple components

Thank you so much!

**Comment by jaames** [comment]
Hi! Thank you!

I think I can do something like this:

```js
const picker = new iro.ColorPicker('#color-picker', {
  ...
  layout: [
    {
       component: iro.Slider,
       options: {
         ...
         id: 'some unique component id'
       }
    }
  ]
});

picker.on('input:end', function(color, originId) {
  if (originId === 'some unique component id') {
    // event comes from the Slider component
  }
});
```
Do you think that would work for you for now? I will probably do a larger overhaul of iro's event system at some point to make it a bit nicer :)

**Comment by Aircoookie** [comment]
That would be an awesome capability to have, thank you! 😄

**Comment by jaames** [comment]
Got it, I'll push it with the next release :)

**Comment by jaames** [comment]
should be available in the latest release (5.3.0)
